services/dynamic-scheduler/tests/unit/conftest.py

```diff
@@ -0,0 +1,29 @@
+from collections.abc import Callable
+from copy import deepcopy
+
+import pytest
+from models_library.api_schemas_dynamic_scheduler.dynamic_services import (
+    DynamicServiceStart,
+    DynamicServiceStop,
+)
+from models_library.projects_nodes_io import NodeID
+
+
+@pytest.fixture
+def get_dynamic_service_start() -> Callable[[NodeID], DynamicServiceStart]:
+    def _(node_id: NodeID) -> DynamicServiceStart:
+        dict_data = deepcopy(DynamicServiceStart.Config.schema_extra["example"])
+        dict_data["service_uuid"] = f"{node_id}"
+        return DynamicServiceStart.parse_obj(dict_data)
+
+    return _
```

Comment:
I thought you did not like using `_` in this way ? ;-) I am fine with it

Reply:
Sorry, I think is exactly the same. It is a variable whose name is not relevant and then you call it `_`.  It is not because the variable is not used or to refer to the latest result, as you pointed out as main uses of `_`.